playwright.config.ts

```diff
@@ -15,7 +15,7 @@ function getCurrentBranch(): string {
 }
 
 // Determine current branch and assign it to a v
-let currentBranch = getCurrentBranch();
+const currentBranch = getCurrentBranch();
```

Comment:
Why was this updated to a const?

Reply:
I believe it was ES Lint that yelled at me to convert it to a const @shashilo. I can convert if needed.